lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -153,37 +204,75 @@ def process_input_until_redraw_required
         # rubocop:disable Style/WhenThen,Layout/SpaceBeforeSemicolon
         def wait_for_user_input
           char = read_char
+          @last_char = char
+
+          case char
+          when ESC      ; @state = :esc and return # Handle escape sequence
```

Comment:
Why was the `ESC` logic changed? Does it still work? Looks like if I type `ESC+ESC`, it will keep it in "esc" state.

Reply:
I hadn't considered that particular code path.  It seemed to only be intended to handle the up/down arrow keys (since those trigger an escape sequence `\e[A`).

I'll change that back.  I wasn't a huge fan of having the same code three times in the state logic, but now I'm wondering if it isn't more clear what's intended that way.